devtools/conda-envs/test_env.yaml

```diff
@@ -0,0 +1,13 @@
+name: openfe-notebooks
```

Comment:
I'm always slightly worried stuff like this will get out of sync when you have duplicates. Is there a way we can squash them to the same location?

Reply:
Yes I agree, they become out of sync and become a footgun for users and developers. When I was playing around with this earlier, I had use openeye in the CI, which couldn't run on mybinder, so it made sense for the CI and mybinder to be different, but since we don't need/want openeye, then it makes sense to have a single point of truth for the env.